apps/settings/js/config/require.js

```diff
@@ -123,7 +123,9 @@ require.config({
         'modules/bluetooth/version_detector',
         'modules/app_storage',
         'modules/battery',
-        'modules/wifi_context'
+        'modules/wifi_context',
+        'modules/bluetooth/bluetooth_v1',
```

Comment:
These two lines are not required because they are not in the dependent list of `LowPriorityItems`.